lib/client/index.js

```diff
@@ -76,7 +76,20 @@ client.init = function init (callback) {
       console.log('Already tried to get settings after auth, but failed');
     } else {
       client.settingsFailed = true;
-      language.set('en');
+
+      // detect browser language
+      var lang = (navigator.language || navigator.userLanguage).toLowerCase();
```

Comment:
For language, you should check the client preferences on the language picked in the UI in the main view. Nightscout explicitly does not use the browser language detection

Reply:
Please see https://github.com/nightscout/cgm-remote-monitor/pull/4860/files#diff-5f3d9018a260bb33cee03e0c67fdf95bR81